### PR TITLE
bpo-28682: Added support for bytes paths in os.fwalk().

### PR DESCRIPTION
**Pull request by serhiy-storchaka**

### CODE REVIEW

File: Lib/test/test_os.py [hunk 1]
```diff
@@ -1091,6 +1094,26 @@ def walk(self, top, **kwargs):
             bdirs[:] = list(map(os.fsencode, dirs))
             bfiles[:] = list(map(os.fsencode, files))
 
+@unittest.skipUnless(hasattr(os, 'fwalk'), "Test needs os.fwalk()")
+class BytesFwalkTests(FwalkTests):
+    """Tests for os.walk() with bytes."""
+    def setUp(self):
+        super().setUp()
+        self.stack = contextlib.ExitStack()
```
Comment (vstinner) [inline Lib/test/test_os.py]: What is the purpose of self.stack?

Comment (serhiy-storchaka) [reply Lib/test/test_os.py]: It is not needed. This is just a copy from BytesWalkTests. But in BytesWalkTests it is not needed either. It was used in the past for silencing deprecation warnings about using Windows bytes API.

**Review by brettcannon — APPROVED** [review]

File: Lib/test/test_os.py (hunk 1 above)
Comment (vstinner) [reply Lib/test/test_os.py]: If it's not used, please remove it. Remove it also from BytesWalkTests.

Comment (vstinner) [reply Lib/test/test_os.py]: oh sorry, you already did it, thanks :-)

**Pull request merged**